audit-triplestore-web/src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ jms.fcrepoEndpoint=${fcrepo.jms.endpoint:topic:fedora}
 jms.brokerUrl=failover:(tcp://${fcrepo.jms.host:localhost}:${fcrepo.dynamic.jms.port:61616})?startupMaxReconnectAttempts=1&initialReconnectDelay=1
 triplestore.baseUrl=${fcrepo.audit.triplestore.baseUrl:localhost:3030/test/update}
 event.baseUri=${fcrepo.audit.baseUri:http://example.com/event}
+audit.container=${fcrepo.audit.container:/audit}
```

Comment:
good idea. The PredicateBuilder class has an `or` function for this very thing:

    filter(not(or(<Expression1>, <Expression2>)))
      .to("destination")